### PR TITLE
Add Test & Solution Concat_args

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: tests/concat2args_test/main.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"github.com/01-edu/go-tests/lib/challenge"
+)
+
+func main() {
+	args := [][]string{
```
Comment (Hamzaelkhatri) [inline tests/concat2args_test/main.go:8]: add empty args

File: tests/concat2args_test/main.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		{"Abc", "Bcd"},
+		{"Hello", " World!"},
+		{"talented ", "student"},
+		{"A", "1"},
+		{"Hello", "World"},
+		{"he","llo"},
+		{"1", "93"},
+		{" ", "		"},
+		{"1 + ", "9 = 10"},
```
Comment (Hamzaelkhatri) [inline tests/concat2args_test/main.go:17]: add more than 3 args too

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

**Review by Hamzaelkhatri — APPROVED** [review]
LGTM!

File: tests/concat2args_test/main.go (hunk 2 above)
Comment (MSilva95) [reply tests/concat2args_test/main.go:18]: add This as well:
```
{""},
{},
```

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: tests/concat2args_test/main.go (hunk 2 above)
Comment (zainabdnaya) [reply tests/concat2args_test/main.go:19]: Done

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**